2020/day2/solution.py

```diff
@@ -0,0 +1,28 @@
+import re
+password_re = re.compile(r"(\d+)-(\d+) (\w): (\w+)")
```

Comment:
Consider using named capture groups - e.g. `r"(?P<first>\d+)-(\d+) (\w): (\w+)"

Reply:
oh, i didn't know I could name them.  That's cool.  Much better than just remembering  what group 1-5 is